### DISCUSSION TITLE
Please help to build this project on Windows

### DISCUSSION THREAD

**Discussion by McRush** [PotatoSpudowski/fastLLaMa#26] [Q&A]
Hello!
I find this project really cool. FastLLama has separate functions for prompt ingesting and text generation unlike other solutions. The ability to save and load model states is also very nice. I really would like to try it.
Unfortunately, now I'm bound to my Windows workflow. I would consider using WSL or install Linux, but building for Windows would be a preferable way for now.

I'm not familiar with C/C++, so I just tried to generate VisualStudio files from CMake and then compile them. Miracle didn't happen and compilation failed.
I suspect that some preparation is necessary here. Maybe I shouldn't use VS at all.
If somebody is ready to provide any instructions, I would very very appreciate it.

**Comment by McRush** [comment]
[cmake_vs_log.txt](https://github.com/PotatoSpudowski/fastLLaMa/files/11089670/cmake_vs_log.txt)
Here is CMake output if anybody curious.

I will try building with CodeBlocks/MinGW soon. Let's see if there will be different result.

**Comment by PotatoSpudowski** [comment]
Hi, 
Sorry for the late reply. I was a bit pre occupied for a few days. I have not tried this on windows yet. I will try running it and will add the instructions. Please give me sometime. Really appreciate the kind words!
